site/specs-source/voice.json

```diff
@@ -3250,6 +3250,191 @@
             "type": "string"
           }
         }
+      },
+      "CallCallback": {
```

Comment:
Are my eyes tricking me or `recordingId` is missing here?

Reply:
It's on line 3314

Reply:
🙈